II.pass-tests/08-get-average/get-average.js

```diff
@@ -1,3 +1,9 @@
 // the input is an array of numbers and strings
 // return the average of all the numbers
-// be sure to exclude the strings
\ No newline at end of file
+// be sure to exclude the strings
+
+module.exports = function average(numbers) {
+  var arrNumbers = numbers.filter(number => Number.isInteger(number));
```

Comment:
Nice! Can you think of a case where this *wouldn't* give the answer you expected?